src/Pos/Lrc/Worker.hs

```diff
@@ -217,14 +218,17 @@ richmenComputationDo
     => EpochIndex -> [LrcConsumer m] -> m ()
 richmenComputationDo epochIdx consumers = unless (null consumers) $ do
     total <- GS.getEffectiveTotalStake
+    logDebug $ "Effective total stake: " <> show total
```

Comment:
Why did you use `show`? AFAIU, `total` is `Coin` and it looks better when formatted based on `Buildable`. You could use `pretty`, for example.